pseudocode.md

```diff
@@ -1,6 +1,19 @@
 # Hangman (Pseudocode)
 
 ```
-Write pseudocode here, within the tripple backticks.
+get user input for the theme
+get user input for the word (based on the theme)
+clear the screen
+
+while player has not won *and* incorrect guesses are less than 10
+  print (10 - incorrect guesses) guesses remaining
```

Comment:
This might be a helpful heads-up on top of having the hanging platform ascii art.